pkg/metrics/metrics.jsx

```diff
@@ -1622,7 +1640,23 @@ class MetricsHistory extends React.Component {
 
         const isBeibootBridge = cmdline?.includes("ic# cockpit-bridge");
 
-        this.setState({ packagekitExists, isBeibootBridge });
```

Comment:
I think it makes sense to keep this, as determining these two are quick. Reading the journal may take several seconds.